packages/frontend/src/contexts/game/actions/index.js

```diff
@@ -9,3 +9,13 @@ export const flipTile = (tile) => ({
   type: types.FLIP_TILE,
   payload: { tile },
 });
+
+export const resetTiles = (tiles) => ({
+  type: types.RESET_TILES,
+  payload: { tiles },
+});
+
+export const handleMatch = (ids) => ({
+  type: types.HANDLE_MATCH,
+  payload: { ids },
```

Comment:
Could the payload for this action be `{ tiles }` as well?

Reply:
Since the `HANDLE_MATCH` action isn't part of this ticket, this could probably be pushed to another ticket with that logic.